python/tskit/trees.py

```diff
@@ -5522,6 +5522,32 @@ def genetic_relatedness(
         :ref:`polarised <sec_stats_polarisation>`,
         and :ref:`return value <sec_stats_output_format>`.
 
+        What is computed depends on ``mode``:
+
+        "site"
+            Number of pairwise allelic matches between two samples relative to
+            the rest of the samples, per unit of chromosome length. To be
+            precise, let `m(a,b)` denote the total number of pairwise allelic
+            matches between samples `a` and `b`, and let `C` and `D` be
+            independently chosen individuals from the sample. Then, this
+            computes `E[m(a,b) - m(a,C) - m(b,D) + m(C,D)]`. If
+            ``proportion=True``, this corresponds to :math:`K_{c0}` in
+            `Doug & Balding (2014) <https://www.nature.com/articles/nrg3821>`_.
+
+        "branch"
+            Total area of branches ancestral to two samples relative to
+            the rest of the samples. To be precise, let `S(a,b)` denote the
+            shared branch area between samples `a` and `b`, and let `C` and `D`
+            be independently chosen individuals from the sample. Then, this
+            computes `E[S(a,b) - S(a,C) - S(b,D) + S(C,D)]`.
+
+        "node"
+            Total number of nodes ancestral to two samples relative to
+            the rest of the samples. To be precise, let `N(a,b)` denote the
+            number of nodes ancestral to samples `a` and `b`, and let `C` and
+            `D` be independently chosen individuals from the sample. Then, this
+            computes `E[N(a,b) - N(a,C) - N(b,D) + N(C,D)]`.
```

Comment:
A node statistic returns, for each node, the sum of the summary function applied to the split induced by the branch above that node, divided by window length. Here, the summary function gives the number of pairs that both inherit from the node, so this is something like "for each node n, let N(a,b) denote the proportion of the window over which a and b are both descendants from n..."

Reply:
OK wow I was completely off before! I think I get it now... Strictly speaking, do we want N(a,b) = \sum_(u,v) n(u,v) where n(u,v) is the proportion of the window over which a and b are both descendants from n? Where the sum is over all pairs of nodes (u,v) where u \in a and v \in b.
And similarly for the branch stat? (since branches ancestral to sample sets is ambiguous)